torch/_refs/nn/functional/__init__.py

```diff
@@ -419,6 +420,32 @@ def hinge_embedding_loss(
     return _apply_loss_reduction(loss, reduction)
 
 
+# TODO: Cannot register as decomp here due to incompatible signatures between
+# the Python frontend and ATen:
+# https://github.com/pytorch/pytorch/issues/83931
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("input", "target"),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+)
+def huber_loss(
```

Comment:
do you need `out` wrapper here?

Reply:
Interestingly `nn.functional.huber_loss` doesn't support an `out` parameter, which is a UX consistency bug:

https://github.com/pytorch/pytorch/blob/d52452b3d1ff14a87e1d79c8d8bd67f0f074e6e6/torch/nn/functional.py#L3199

We should probably scrub and correct these UX consistency issues together. I made a note to follow-up.

Reply:
no, i think. the yaml file has the out overload, but the opinfo and the function signature say that `out` is not supported

Reply:
Note that this is an issue with many of the loss functions really: the function has an out variant on ATen, but it's not exposed on the `functional` API.

Reply:
> We should probably scrub and correct these UX consistency issues together. I made a note to follow-up.

note: i have a wip local branch that allows testing for `out` support consistency between regular pytorch ops and primtorch refs. it needs some clean up, but i'm planning to submit once the ref alias checking stack is merged

Reply:
Note that it's fine if a reference has the `out=` behaviour but the original operation doesn't. Also, we already have a number of skips/errors for `test_out` in core, so I'd expect that, if anything, the references are more correct than the functions in core. It may be a bit tricky to write a test that's not too flaky and that test all these issues.